Formula/d/dipc.rb

```diff
@@ -0,0 +1,17 @@
+class Dipc < Formula
+  desc "Convert your favorite images/wallpapers with your favorite color palettes/themes"
+  homepage "https://github.com/doprz/dipc"
+  url "https://github.com/doprz/dipc/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "dd98bf2eea8e97dfaeb8d4e0a991a732e35bf71e1b9bdf0045fdad80e0c0d319"
+  license any_of: ["MIT", "Apache-2.0"]
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system bin/"dipc", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate teformula-surferath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
My cli tool takes an image as an input and outputs and image. How would you suggest we proceed with this to provide a good test?

Reply:
@chenrui333 I've added a proper test and this PR is now ready for review. Thank you for your feedback!